File: api/generateBlog.py
# api/generateBlog.py
import json

def handler(request):
    """
    Vercel Python Serverless Function
    POST /api/generateBlog
    """
    data = request.json() or {}
    kw = data.get("keyword", "")

    return {
        "statusCode": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps({
            "title": f"{kw} を極めるバイオハック入門",
            "outline": "- 導入\n- 問題提起\n- 解決策\n- まとめ"
        })
    }

File: api/generateBlog.ts
// api/generateBlog.ts
import type { VercelRequest, VercelResponse } from '@vercel/node';

export default function handler(req: VercelRequest, res: VercelResponse) {
  const { keyword = '' } = req.body ?? {};
  res.status(200).json({
    title: `${keyword} を極めるバイオハック入門`,
    outline: '- 導入\n- 問題提起\n- 解決策\n- まとめ'
  });
}

